src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.Scanner;
+
+public class Ui {
+
+    protected Scanner sc;
```

Comment:
Should this Scanner be named scanner as it is in the scope of the whole class? I refer to the point below in the module's coding standard:
"Variables with a large scope should have long names, variables with a small scope can have short names. "